network/peermgr.go

```diff
@@ -39,8 +39,8 @@ func newPeerMgr(ctx context.Context, h lp2phost.Host,
 	b := &peerMgr{
 		ctx:            ctx,
 		bootstrapAddrs: conf.BootstrapAddrInfos(),
-		minConns:       conf.MinConns,
-		maxConns:       conf.MaxConns,
+		minConns:       conf.ScaledMaxConns(),
+		maxConns:       conf.ScaledMinConns(),
```

Comment:
```suggestion
		maxConns:       conf.ScaledMaxConns(),
```

and max for this one.